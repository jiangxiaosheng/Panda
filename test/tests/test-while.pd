func main() {
	var sum = 0
    var i = 0

    while i < 10 {
        sum += i
        i += 1
		if 1 == 1 {
			break
		}
    }

}