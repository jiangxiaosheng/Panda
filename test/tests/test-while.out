Sementically checked program: 


void main() {
var sum: int(int : 0)
var i: int(int : 0)
while ((bool : (int : i) < (int : 10))) {
(int : sum += (int : i))
(int : i += (int : 1))
}
}

